client/app/validations/saveable-applicant-form.js

```diff
@@ -26,7 +26,8 @@ export default {
       message: 'Text is too long (max {max} characters)',
     }),
     validateFormat({
-      type: 'email',
+      // type: 'email',
+      regex: /[0-9]{10}/,
```

Comment:
Wait why does email have a phone number regex?

Reply:
Whoops! I was testing around and forgot to remove that